distributed/tests/test_spec.py

```diff
@@ -1,18 +1,22 @@
 from distributed.deploy.spec import ProcessInterface
+from distributed.utils_test import gen_test
 
 
-def test_address_default_none():
-    p = ProcessInterface()
-    assert p.address is None
+@gen_test()
+async def test_address_default_none():
+    async with ProcessInterface() as p:
```

Comment:
ProcessInterface calls `asyncio.Lock()` and `asyncio.Event()` in the constructor - which call `get_event_loop()` in py3.8-3.9